grumblr/app/assets/javascripts/templates/grumbls/grumblsListView.js

```diff
@@ -0,0 +1,18 @@
+var GrumblsListView = Backbone.View.extend({
+  el: "#grumble-list",
+
+  initialize: function(){
+    console.log("created grumbl list view" );
+    this.listenTo(this.collection, "add", this.renderAll);
```

Comment:
When adding to a collection, we don't need to re-render the whole collection `renderAll`. Instead, you can do `renderOne` to render the newly added model.